resources/roca_main.brs

```diff
@@ -0,0 +1,41 @@
+sub main()
+    basePath = "tests"
+    files = MatchFiles("pkg:/" + basePath, "*.test.brs")
+    print "# found " files.count() " files"
+    roca = "pkg:/tests/roca.brs"
+
+    rootSuites = []
+    for each file in files
+        path = ["pkg:", basePath, file].join("/")
+        suite = _brs_.runInScope(path, {})
+        if suite <> invalid then rootSuites.push(suite)
+    end for
+
+    focusedCasesDetected = false
+    for each suite in rootSuites
+        focusedCasesDetected = (focusedCasesDetected or suite.__state.transitivelyHasFocusedCases)
+    end for
+
+    totalTests = 0
+    for each suite in rootSuites
+        totalTests += suite.__state.totalCases
+    end for
+
+    tap = tap()
+    tap.version()
+    tap.plan(rootSuites.count())
```

Comment:
Don't we want to pass the `totalTests` here? or where is being used?

Reply:
Surprisingly no!  `tap-parser` seems to want this to be the number of suites.  I think `totalTests` isn't necessary anymore, so I can remove it :)